recipe/meta.yaml

```diff
@@ -49,7 +45,8 @@ test:
     - alembic.util
   requires:
     - pip
-    - python <3.10
+    - python
+    - pytest
   commands:
     - alembic --help
     - pip check
```

Comment:
Missing doc_source_url: https://github.com/sqlalchemy/alembic/tree/main/docs/build

Reply:
Could you add the description too: `Alembic is a lightweight database migration tool for usage with the SQLAlchemy Database Toolkit for Python.` ? See https://alembic.sqlalchemy.org/en/latest/index.html